eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -174,13 +174,19 @@ stages:
                             PackageInfoLocations:
                               - $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo/${{artifact.name}}.json
                             WorkingDirectory: $(System.DefaultWorkingDirectory)
-                            TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
+                            TargetDocRepoOwner: 'Azure'
```

Comment:
We don't have the RepoOwner parameter in the yml

Reply:
Nit: you don't need quotes most times in Yaml. Only if it begins with a number (but is a string, like a version) or a special character.